.gitignore

```diff
@@ -224,3 +224,8 @@ config/configswapper.json
 config/betterendforge/client.json
 config/Mekanism/tiers.toml
 pregen/supporters.json
+schematics/uploaded/
+modernfix/structureCacheV1/
+lightspeed-cache/1.16.5/
+config/appliedenergistics2-client.toml
+config/rubidium-options.json

```

Comment:
Please remove these additions to the .gitignore :)

Reply:
Oh, I forgot this. Fixed in new commits